cypress/integration/lesson14_spec.js

```diff
@@ -0,0 +1,34 @@
+describe('My First Test', () => {
+  before(() => {
+    cy.visit('https://demoqa.com')
```

Comment:
After added base url into cypress.json u will use 
```suggestion
    cy.visit('/')
```

Reply:
Accepted.